src/convert.js

```diff
@@ -2,6 +2,9 @@
 import simplify from './simplify';
 import createFeature from './feature';
 
+const maxMercatorLatitude = 85.05;
+const minVectorLatitude = -maxMercatorLatitude * 8191/8192;
```

Comment:
Why 8191/8192? Probably should add a comment explaining this

Reply:
👍 will add comment if we end up going this route. for now, it's explained in the PR text